### PR TITLE
[Obmin-266] checkbox

### PR DESCRIPTION
**Pull request by AkintsevaElizaveta**
Replaced CheckBox component with component from Storybook

### CODE REVIEW

File: src/main/resources/react/src/components/common/filtration/index.jsx [hunk 1]
```diff
@@ -91,10 +91,11 @@ const Filtration = () => {
             <CheckBoxes title={getTranslatedText('product.sex')}>
               {sexShow.map((item) => (
                 <FormikCheckBox
+                  gap={20}
                   key={item}
                   value={item}
                   type="checkbox"
-                  fontSize="16px"
+                  fontSize={16}
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/common/filtration/index.jsx:98]: sort

File: src/main/resources/react/src/components/common/filtration/index.jsx [hunk 2]
```diff
@@ -106,10 +107,11 @@ const Filtration = () => {
             <CheckBoxes title={getTranslatedText('product.age')}>
               {agesShow.map((item) => (
                 <FormikCheckBox
+                  gap={20}
                   key={item}
                   value={item}
                   type="checkbox"
-                  fontSize="16px"
+                  fontSize={16}
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/common/filtration/index.jsx:114]: the same

File: src/main/resources/react/src/components/common/filtration/index.jsx [hunk 3]
```diff
@@ -122,10 +124,11 @@ const Filtration = () => {
               <Styles.ScrollBar>
                 {receivedClothingSizes.map((item) => (
                   <FormikCheckBox
+                    gap={20}
                     key={item}
                     value={item}
                     type="checkbox"
-                    fontSize="16px"
+                    fontSize={16}
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/common/filtration/index.jsx:131]: and again

File: src/main/resources/react/src/pages/FormRegister/sign-in/index.jsx [hunk 4]
```diff
@@ -7,9 +7,11 @@ import { useNavigate, useLocation } from 'react-router-dom';
 import { NO_SPACE } from 'config';
 import { GoogleSvg } from 'assets/icons';
 import { route } from 'routes/routeConstants';
+import { CheckBox } from '@wolshebnik/obminyashka-components';
+
 import { putUserThunk } from 'store/auth/thunk';
 import { getTranslatedText } from 'components/local/localization';
-import { CheckBox, Button, InputForAuth } from 'components/common';
+import { Button, InputForAuth } from 'components/common';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/FormRegister/sign-in/index.jsx:14]: sort

File: src/main/resources/react/src/pages/FormRegister/sign-up/index.jsx [hunk 5]
```diff
@@ -5,9 +5,10 @@ import { useNavigate } from 'react-router-dom';
 import api from 'REST/Resources';
 import { GoogleSvg } from 'assets/icons';
 import { route } from 'routes/routeConstants';
+import { CheckBox } from '@wolshebnik/obminyashka-components';
 
 import { getTranslatedText } from 'components/local/localization';
-import { CheckBox, Button, InputForAuth } from 'components/common';
+import { Button, InputForAuth } from 'components/common';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/FormRegister/sign-up/index.jsx:11]: sort

File: src/main/resources/react/src/pages/FormRegister/sign-up/index.jsx [hunk 6]
```diff
@@ -74,10 +75,12 @@ const SignUp = () => {
 
             <Extra>
               <CheckBox
-                fontSize="14px"
+                gap={22}
+                fontSize={14}
                 checked={checkbox}
+                style={{ paddingRight: '10px' }}
                 margin="0 0 44px 0"
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/FormRegister/sign-up/index.jsx:82]: sort

File: src/main/resources/react/src/pages/UserInfo/tab_pages/my_profile/children/gender/index.jsx [hunk 7]
```diff
@@ -14,26 +15,27 @@ const Gender = ({ name }) => {
...
-            {getTranslatedText('ownInfo.unselected')}
-          </Styles.LabelGander>
-        </Styles.ChildDiv>
+        <CheckBox
+          type="radio"
+          fontSize={16}
+          onChange={() => helpers.setValue(enumSex.MALE)}
+          checked={value === enumSex.MALE}
+          text={getTranslatedText('ownInfo.boy')}
+        />
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/UserInfo/tab_pages/my_profile/children/gender/index.jsx:24]: space

File: src/main/resources/react/src/pages/UserInfo/tab_pages/my_profile/children/gender/index.jsx [hunk 8]
```diff
@@ -14,26 +15,27 @@ const Gender = ({ name }) => {
...
+        <CheckBox
+          type="radio"
+          fontSize={16}
+          onChange={() => helpers.setValue(enumSex.FEMALE)}
+          checked={value === enumSex.FEMALE}
+          text={getTranslatedText('ownInfo.girl')}
+        />
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/UserInfo/tab_pages/my_profile/children/gender/index.jsx:31]: space

File: src/main/resources/react/src/pages/UserInfo/tab_pages/my_profile/children/gender/index.jsx [hunk 9]
```diff
@@ -14,26 +15,27 @@ const Gender = ({ name }) => {
...
+        <CheckBox
+          type="radio"
+          fontSize={16}
+          onChange={() => helpers.setValue(enumSex.UNSELECTED)}
+          checked={value === enumSex.UNSELECTED}
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/UserInfo/tab_pages/my_profile/children/gender/index.jsx:36]: sort

File: src/main/resources/react/src/components/common/formik/formik-checkbox/index.jsx [hunk 10]
```diff
@@ -37,22 +38,18 @@ const FormikCheckBox = ({
...
-        <span>{text}</span>
-      </Label>
+    <Div>
+      <CheckBox
+        gap={gap}
+        type={type}
+        text={text}
+        name={name}
+        margin={margin}
+        value={value}
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/common/formik/formik-checkbox/index.jsx:48]: sort

**Review by Wolshebnik — APPROVED** [review]

**Review by Mish-hash — APPROVED** [review]

**Review by Voloshkina — APPROVED** [review]

**Review by Viktoria-Serkiz — APPROVED** [review]

**Review by ASGladchenko — APPROVED** [review]

**Pull request merged**